iac-extensions/kubernetes/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,18 @@
+<configuration>
```

Comment:
Do we need that? We are using `LogTesterJUnit5 `

Reply:
I was using it to see logs of the process during debugging, but I guess it can be safely removed now